rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml

```diff
@@ -0,0 +1,31 @@
+title: Gather information about a system using gatherNetworkInfo.vbs script
```

Comment:
`has a title that has not title capitalization. Words: 'information, about, system, using, script'`